agent/discovery/query_fetcher_v1.go

```diff
@@ -424,6 +424,7 @@ RPC:
 }
 
 func (f *V1DataFetcher) ValidateRequest(_ Context, req *QueryPayload) error {
+	f.logger.Debug(fmt.Sprintf("req %+v", req))
```

Comment:
~ Possibly intentional but wasn’t sure this was meant to stay after local testing

Reply:
i removed.